tests/integration_tests/build/test_binary_size.py

```diff
@@ -14,15 +14,15 @@
 SIZES_DICT = {
     "x86_64": {
         "FC_BINARY_SIZE_TARGET": 2367352,
-        "JAILER_BINARY_SIZE_TARGET": 797632,
+        "JAILER_BINARY_SIZE_TARGET": 850224,
         "FC_BINARY_SIZE_LIMIT": 2485719,
-        "JAILER_BINARY_SIZE_LIMIT": 869608,
+        "JAILER_BINARY_SIZE_LIMIT": 892735,
     },
     "aarch64": {
         "FC_BINARY_SIZE_TARGET": 2208040,
-        "JAILER_BINARY_SIZE_TARGET": 775456,
+        "JAILER_BINARY_SIZE_TARGET": 851152,
         "FC_BINARY_SIZE_LIMIT": 2318442,
-        "JAILER_BINARY_SIZE_LIMIT": 845247,
+        "JAILER_BINARY_SIZE_LIMIT": 893709,
```

Comment:
same here

Reply:
Updating values with the correct numbers. Closing this for now.